Exercise1/html/exercise1.html

```diff
@@ -0,0 +1,273 @@
+                        <li class="drop-li"><a href="#">CATEGORY 2</a></li>
+                        <li><a href="#">CATEGORY 3</a></li>
+                      </ul>
+                    </li>
+                    <li><a href="#">GALLERY 4</a></li>
+                  </ul>
+              </li>
+              <li><a href="#">EVENTS</a></li>
+              <li><a href="#">BLOG</a></li>
+              <li><a href="#">CONTACTS</a></li>
+            </ul>
+          </nav>
+          <div class="search">
+            <input type="text">
+            <input type="submit" name="" value="">
+            <i class="fa fa-search"></i>
+          </div>
+        </div>
+      </div>
+    </header>
+    <section class="content">
+      <div class="slide-show">
+        <div class="show-img">
+          <img src="../images/slide3.jpg">
+          <h2><strong>CONSECTETUER</strong><br>STARTING FROM: $299.99</h2>
+          <div class="back-next">
+            <button class="gallery-left">
+              <i class="fa fa-angle-left" aria-hidden="true"></i>
+            </button>
+            <button class="gallery-right">
+              <i class="fa fa-angle-right" aria-hidden="true"></i>
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="feature-space">
+      <h2>FEATURED</h2>
+    </section>
+    <section class="image-show">
+      <div class="one-img"><!-- show-one-img -->
+        <img src="../images/image1.jpg">
+      </div>
+      <div class="one-img"><!-- show-one-img -->
+        <img src="../images/Depositphotos_4474020_original-199x128.jpg">
+      </div>
+      <div class="one-img"><!-- show-one-img -->
+        <img src="../images/Depositphotos_3.jpg">
+      </div>
+      <div class="one-img"><!-- show-one-img -->
+        <img src="../images/image4.jpg">
+      </div>
+      <div class="one-img"><!-- show-one-img -->
+        <img src="../images/images5.jpg">
+      </div>
+    </section>
+    <section class="next-img">
+      <button>
+        <i class="fa fa-angle-right" aria-hidden="true"></i>
+      </button>
+    </section>
+    <section class="some-news">
+      <article class="one-block-new"> <!-- article for one block -->
+        <h5>
+          NEWS<br><span>& EVENTS</span>
+        </h5>
+        <img src="../images/new-event.jpg">
+        <p>Donec sit amet felis orci. Fuscet mollis lectus. Duis in mattis nibh. Mauris tempus, massa non auct consectetur, nulla est.</p>
+        <button type="button-one"><span class="color-one">MORE</span></button>
+      </article>
+      <article class="one-block-new"> <!-- article for one block -->
+        <h5>
+          NEWS<br><span>DESTINATIONS</span>
+        </h5>
+        <img src="../images/new-2.jpg">
+        <p>Donec sit amet felis orci. Fuscet mollis lectus. Duis in mattis nibh. Mauris tempus, massa non auct consectetur, nulla est.</p>
+        <button type="button-one"><span class="color-two">MORE</span></button>
```

Comment:
`<button type="button-one">` to <button type="button">
- Don't have `type="button-one"`